src/mapGenerator.js

```diff
@@ -37,6 +37,13 @@ export default function mapGenerator(start, goal, grid) {
             continue;
         }
 
+        if ((holeX + holeRadius >= grid.width - 1 && holeY + holeRadius >= grid.height - 1) ||
+            (holeX + holeRadius >= grid.width - 1 && holeY - holeRadius <= 0) ||
+            (holeX - holeRadius <= 0 && holeY + holeRadius >= grid.height - 1) ||
+            (holeX + holeRadius <= 0 && holeY - holeRadius <= 0)) {
```

Comment:
each comparison in here is done twice; DRY
case in point, you have a `holeX - holeRadius <= 0` and an invalid `holeX + holeRadius <= 0` right next to it

Reply:
good point 🐇